subjects/descendappendrange/README.md

```diff
@@ -0,0 +1,46 @@
+## descendappendrange
+
+### Instructions
+
+- Write a function that takes an `int` max and an `int` min as parameters. The function should return a slice of `int`s with all the values between the max and min.
+
+- The `Max`must be included, and min must be excluded.
```

Comment:
max instead of Max, variables declared with lowercase. Also both min and max inside "``" or both without.